h2o-py/h2o/h2o.py

```diff
@@ -515,12 +515,14 @@ def load_grid(grid_file_path):
     return get_grid(response["name"])
 
 
-def save_grid(grid_directory, grid_id):
+def save_grid(grid_directory, grid_id, export_cross_validation_predictions=False):
     """
     Export a Grid and it's all its models into the given folder
 
     :param grid_directory: A string containing the path to the folder for the grid to be saved to.
-    :param grid_id: A chracter string with identification of the Grid in H2O. 
+    :param grid_id: A character string with identification of the Grid in H2O.
+    :param export_cross_validation_predictions: A boolean flag indicating whether the models exported from the grid should be saved
```

Comment:
maybe also document that default is False

Reply:
good idea, thank you - added to affected methods